internal/models/user.go

```diff
@@ -65,3 +67,18 @@ func (u *User) CreateUser(db *gorm.DB) error {
 
 	return nil
 }
+
+func (u *User) GetSeedUsers(db *gorm.DB) ([]User, error) {
+	var users []User
+
+	if err := db.Preload("Profile").Preload("Products").Preload("Organisations").Limit(2).Find(&users).Error; err != nil {
+		return users, err
+	}
```

Comment:
create a generic function in repository that can help with these multiple preloads

Reply:
done sir